src/librustc_mir/hair/pattern/_match.rs

```diff
@@ -1767,7 +1767,7 @@ pub fn is_useful<'p, 'a, 'tcx>(
         && !v_constructors.iter().any(|ctor| ctor.is_wildcard())
     {
         debug!("is_useful - shortcut because declared non-exhaustive");
-        // FIXME(Nadrieril): this behavior is incorrect; raise an issue about it.
+        // FIXME(#65157)
```

Comment:
Nice catch. It'd be clearer to squash this into the commit that introduced the `FIXME`.